### DISCUSSION TITLE
Add possibility for customizing password generator

### DISCUSSION THREAD

**Discussion by masc88** [dani-garcia/vaultwarden#5083] [Ideas]
It would be great, if one could modify the special characters in the password generator to own needs, i.e. the feature from KeepassXC

![image](https://github.com/user-attachments/assets/cb662192-5a5d-44dc-ae43-aac7e7616ee5)

Currently, one can just enable special charaters, which include $ % and ^. Those characters might cause issues on some (especially older) systems, so one has to remove/adapt them by hand, if not wanted.

**Comment by BlackDex** [comment]
This is a client side feature, and clients are not maintained by Vaultwarden but by Bitwarden.
Such feature requests should be placed in there repo/community forum.
